packages/runner/src/run.ts

```diff
@@ -240,68 +246,83 @@ export async function runSuite(suite: Suite, runner: VitestRunner) {
     suite.result.state = 'todo'
   }
   else {
-    try {
-      beforeAllCleanups = await callSuiteHook(suite, suite, 'beforeAll', runner, [suite])
+    let retry = suite.repeats
```

Comment:
By default this seems to be `undefined` so no tests are run. Should it default to `|| 1`?

Or maybe to `|| 0` and in the loop condition below:

```diff
-retryCount < retry!
+retryCount <= retry
```